src/Raven.Server/Documents/Expiration/ExpirationStorage.cs

```diff
@@ -246,98 +235,146 @@ private static bool HasPassed(BlittableJsonReaderObject metadata, string metadat
             return false;
         }
 
-        public int DeleteDocumentsExpiration(DocumentsOperationContext context, Dictionary<Slice, List<(Slice LowerId, string Id)>> expired, DateTime currentTime)
+        public class ExpiredDocumentInfo
```

Comment:
The `Expired` part may mislead that the document has already expired. 
We should name it `DocumentExpirationInfo`, `ScheduledExpirationInfo` or similarly.

Reply:
https://github.com/ravendb/ravendb/pull/18542/files#r1606473131
We create this object for the document that didn't expire yet here

Reply:
done